x/rollapp/keeper/liveness_test.go

```diff
@@ -4,31 +4,33 @@ import (
 	"flag"
 	"fmt"
 	"slices"
-	"testing"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	keepertest "github.com/dymensionxyz/dymension/v3/testutil/keeper"
-	"github.com/dymensionxyz/dymension/v3/x/rollapp/keeper"
-	"github.com/dymensionxyz/dymension/v3/x/rollapp/types"
 	"github.com/dymensionxyz/sdk-utils/utils/urand"
 	"github.com/stretchr/testify/require"
 	"pgregory.net/rapid"
+
+	"github.com/dymensionxyz/dymension/v3/x/rollapp/keeper"
+	"github.com/dymensionxyz/dymension/v3/x/rollapp/types"
 )
 
 // Storage and query operations work for the event queue
-func TestLivenessEventsStorage(t *testing.T) {
+func (suite *RollappTestSuite) TestLivenessEventsStorage() {
 	_ = flag.Set("rapid.checks", "50")
 	_ = flag.Set("rapid.steps", "50")
 
 	rollapps := rapid.StringMatching("^[a-zA-Z0-9]{1,10}$")
 	heights := rapid.Int64Range(0, 10)
 	isJail := rapid.Bool()
-	rapid.Check(t, func(r *rapid.T) {
-		k, ctx := keepertest.RollappKeeper(t)
+	rapid.Check(suite.T(), func(r *rapid.T) {
+		suite.SetupTest()
+
+		k, ctx := suite.App.RollappKeeper, suite.Ctx
 		model := make(map[string]types.LivenessEvent) // model actual sdk storage
```

Comment:
this test is neither long running nor does it run when you run other tests, so no need to change

Reply:
done